homeassistant/components/zwave_js/strings.json

```
@@ -150,6 +150,10 @@
     "invalid_server_version": {
       "title": "Newer version of Z-Wave JS Server needed",
       "description": "The version of Z-Wave JS Server you are currently running is too old for this version of Home Assistant. Please update the Z-Wave JS Server to the latest version to fix this issue."
+    },
+    "dry_fan_presets_deprecation": {
+      "title": "{preset_name} preset mode will be removed: {entity_id}",
+      "description": "You are getting this issue because you are using the {preset_name} preset mode in your entity `{entity_id}`.\n\nDry and Fan preset modes are deprecated and will be removed in a future version of Home Assistant. To fix this issue, please update your automations to use the corresponding Dry and Fan **HVAC modes** instead."
```

Issue won't automatically be fixed, the user will have to dismiss it once they've made the changes

```suggestion
      "description": "You are using the {preset_name} preset mode in your entity `{entity_id}`.\n\nDry and Fan preset modes are deprecated and will be removed in a future version of Home Assistant. Please update your automations to use the corresponding Dry and Fan **HVAC modes** instead. You can then manually repair this issue to resolve it."
```

---

Updated text.